src/controllers/beneficial.owner.gov.controller.ts

```diff
@@ -1,21 +1,28 @@
 import { NextFunction, Request, Response } from "express";
 import { logger } from "../utils/logger";
 import * as config from "../config";
-import { ApplicationData, ApplicationDataType, beneficialOwnerGovType } from "../model";
-import { getApplicationData, mapFieldsToDataObject, prepareData, setApplicationData } from "../utils/application.data";
+import { ApplicationData, ApplicationDataType } from "../model";
+import { getApplicationData, mapDataObjectToFields, mapFieldsToDataObject, prepareData, setApplicationData } from "../utils/application.data";
 import { AddressKeys, BeneficialOwnerNoc, HasSamePrincipalAddressKey, InputDateKeys, IsOnSanctionsListKey, NonLegalFirmNoc } from "../model/data.types.model";
 import { PrincipalAddressKey, PrincipalAddressKeys, ServiceAddressKey, ServiceAddressKeys } from "../model/address.model";
 import { StartDateKey, StartDateKeys } from "../model/date.model";
+import { BeneficialOwnerGovKey, BeneficialOwnerGovKeys } from "../model/beneficial.owner.gov.model";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `GET ${config.BENEFICIAL_OWNER_GOV_PAGE}`);
 
     const appData: ApplicationData = getApplicationData(req.session);
 
+    const boGov = appData[BeneficialOwnerGovKey];
```

Comment:
boGov will be a list of boGov objects, so will this only display the fist one? how do we know we get the correct one from teh list?

Reply:
Hi Mark, good point, until we do not have the update and delete logic we do not visualise(GET) previous data for BOs and MOs

Reply:
I created this POC [here](https://github.com/companieshouse/overseas-entities-web/compare/feature/roe-147-simplify-adding-types-of-beneficial-owners) for updating and deleting BOs and MOs

Reply:
But as you have suggested probably better remove the logic for the flat address fields right now